src/components/SecondPage/SecondPageModal/DeleteModalBox.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import history from "../../../history";
+import "./modal.css";
+
+const SecondModal = (props) => {
+  return ReactDOM.createPortal(
```

Comment:
Awesome I can see you are using portals here.  
Just make sure you understand it.